ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly-4.12-upgrade-from-aro.yaml

```diff
@@ -31,22 +31,13 @@ base_images:
     name: "4.12"
     namespace: ocp
     tag: tools
-  upi-installer:
```

Comment:
I checked again `cucushift-installer-rehearse-azure-aro` install workflow, `aro-provision-vnet` and `aro-deprovision` are also using `from: upi-installer` image, so I think we have to keep the `upi-installer` base image here, unless you also update both of the steps to use a tagname of upi-installer.